external-docs/docs/quick-start/qube config.md

```diff
@@ -0,0 +1,239 @@
+| `codelist`       | Link to an existing code list (Optional) (Advanced)          | *none*                                                       |
+
+## Observations
+
+Observations are the most important component of a CSV-W data set. Observation columns can have measures and units defined against them to obviate the need for separate unit and measure columns in a single unit/measure data set.
+
+| **field name** | **description**                                              | **default value** |
+| -------------- | ------------------------------------------------------------ | ----------------- |
+| `data_type`    | The data type of the observations. This can only be decimal or integer. (Optional) | *decimal*         |
+| `unit`         | The unit for this observation column; this can a uri to an existing unit, or a dictionary containing a new or extended existing unit. If there is a unit column this value must not be provided. (Optional) | *none*            |
+| `measure`      | The measure for this observation column; this can be a uri to an existing dimension, or a dictionary containing a new or extneded existing measure. If there is a measure column this key must not be provided. (Optional) | *none*            |
+
+## Measure and Unit Columns
+
+Measure and unit columns are treated slightly differently to dimension, attribute, and observation columns. Measure and unit columns contain references to discrete units and measures. In both cases by defining `"type": "measure column"` or `"type": "unit column"` provides the same behaviour. Do not put measures in units columns and units and measure columns.
+
+| **field name** | **description**                                              | **default value** |
+| -------------- | ------------------------------------------------------------ | ----------------- |
+| `type`         | The type of the column, provide `"measure colum"` for the measure column type or `"unit column"` for the unit column (Required) | *dimension*       |
+| `values`       | If basic units/measures are desired, a boolean value of `true` is used to signify to csvcubed to create units/measures from values of this column; otherwise values is a dictionary which defines the units/measures using the notion from [Measures and Units](#Measures and Units) | `true`            |
+
+# Measures and Units
+
+Measure and units can be attached to a Measure and Unit Column
+
+## Measures
+
+Measures have no unique configuration options.
+
+| **field name**   | **description**                                              | **default value**                                            |
+| ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `type`           | The type of the column (Required)                            | *dimension*                                                  |
```

Comment:
>  `type`           | The type of the column (Required)

An individual measure or unit is not associated with a column, and shouldn't have a `type` property since they're always places in locations where it is unambiguous that they are either a unit or measure.